package.json

```diff
@@ -56,9 +56,11 @@
     "concurrently": "^5.2.0",
     "escape-string-regexp": "^4.0.0",
     "express": "^4.17.1",
+    "firebase": "^9.6.9",
```

Comment:
I believe the plan of record was to load the Firebase and GreenSock Animation Platform libraries only on the `/100/` page from a CDN URL via a `<script>` tag, to ensure that only the page that used them will have to download them.

With the currently implemention in this PR, code for `firebase` and `gsap` ends up being loaded and run as part of the `main.js` bundle that's used on each page.

Reply:
Replaced with CDN and removed from package.json.